scripts/start

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+
+./scripts/start-deps
```

Comment:
This script serves no purpose other than delegating to other scripts. The more indirection the less obvious it is to see what is going on.

Reply:
The start script? It serves the purpose of running multiple servers. We can put all the commands here but I think it's helpful to have them separate so that they can be run individually if needed